source/features/convert-release-to-draft.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import LoadingIcon from '../github-helpers/icon-loading';
+import {getRepoURL} from '../github-helpers';
+
+async function convertToDraft({delegateTarget: draftButton}: delegate.Event<MouseEvent, HTMLButtonElement>): Promise<void> {
+	try {
+		await api.expectToken();
+		draftButton.textContent = 'Converting...';
+		draftButton.append(<LoadingIcon className="ml-2" width={16}/>);
+
+		const tagName = location.pathname.split('/').pop()!;
+		const {id: releaseID}: Record<string, number> = await api.v3(`repos/${getRepoURL()}/releases/tags/${tagName}`);
+		const response = await api.v3(`repos/${getRepoURL()}/releases/${releaseID}`, {
+			method: 'PATCH',
+			body: {
+				draft: true
+			}
+		});
+
+		location.pathname = [getRepoURL(), 'releases', 'edit', response.tag_name].join('/');
+	} catch (error) {
+		draftButton.textContent = 'Convert Failed! See console for details';
+		throw error;
+	}
+}
+
+function init(): void | false {
+	const editButton = select('.BtnGroup a[href*="releases/edit"]')!;
+	if (!editButton || select.exists('.label-draft')) {
+		return false;
+	}
+
+	editButton.after(<a className="btn BtnGroup-item text-orange rgh-convert-draft">Convert to draft</a>);
```

Comment:
I don't think I've ever seen an orange button anywhere on GH
```suggestion
	editButton.after(<a className="btn BtnGroup-item rgh-convert-draft">Convert to draft</a>);
```

Reply:
I was following the was following the way the OP (Original Poster) requested it.

Should there be any color?